tests/amr/data/particles/refine/test_tag_strategy.h

```diff
@@ -26,41 +26,59 @@
 using namespace PHARE::core;
 using namespace PHARE::amr;
 
+template<std::size_t dimension, typename Box>
+std::array<int, 3> boxBoundsLower(Box const& box)
+{
+    return {box.lower(dirX), dimension > 1 ? box.lower(dirY) : 0,
+            dimension > 2 ? box.lower(dirZ) : 0};
+}
+template<std::size_t dimension, typename Box>
+std::array<int, 3> boxBoundsUpper(Box const& box)
+{
+    return {box.upper(dirX), dimension > 1 ? box.upper(dirY) : 0,
+            dimension > 2 ? box.upper(dirZ) : 0};
+}
+
+
 template<std::size_t dimension>
-std::vector<Particle<dimension>> loadCell(int iCell)
+std::vector<Particle<dimension>> loadCell(int iCellX, int iCellY, int iCellZ)
 {
+    std::array<int, 3> _3diCell = {iCellX, iCellY, iCellZ};
+
     float middle = 0.5;
     float delta  = 0.30f;
 
-    Particle<dimension> tmpParticle;
+    Particle<dimension> particle;
     std::vector<Particle<dimension>> particles;
 
-    tmpParticle.weight = 1.;
-    tmpParticle.charge = 1.;
-    tmpParticle.v      = {{1.0, 0.0, 0.0}};
+    particle.weight = 1.;
+    particle.charge = 1.;
+    particle.v      = {{1.0, 0.0, 0.0}};
+    particle.delta.fill(middle);
 
-    tmpParticle.iCell[dirX] = iCell;
+    for (size_t i = 0; i < dimension; i++)
```

Comment:
can't we just di particle.iCell = {iCellX, iCellY, iCellZ}; ?

Reply:
will check

Reply:
![image](https://user-images.githubusercontent.com/30491/86921629-b86b0f80-c12b-11ea-961a-b3571a711cc5.png)